flag.go

```diff
@@ -134,10 +134,15 @@ type FlagSet struct {
 	// a custom error handler.
 	Usage func()
 
+	// SortFlags is used to indicate, if user wants to have sorted flags in
+	// help/usage messages.
+	SortFlags bool
+
 	name              string
 	parsed            bool
 	actual            map[NormalizedName]*Flag
 	formal            map[NormalizedName]*Flag
+	order             []NormalizedName
```

Comment:
I'm not sure I like how this works. I'm wondering if it should just be a []*Flag and be used other places...   But I'm unsure...